go.mod

```diff
@@ -54,7 +54,8 @@ require (
 	github.com/elazarl/goproxy v0.0.0-20190421051319-9d40249d3c2f // indirect
 	github.com/elazarl/goproxy/ext v0.0.0-20190421051319-9d40249d3c2f // indirect
 	github.com/emicklei/go-restful v2.6.0+incompatible
-	github.com/envoyproxy/go-control-plane v0.8.4
+	github.com/envoyproxy/go-control-plane v0.8.5
```

Comment:
add `go mod vendor`

Reply:
You should revert this go.mod change and do `go get github.com/envoyproxy/go-control-plane@v0.8.5` only.

Reply:
Will do. Thank for the instruction!